diesel/src/migration/mod.rs

```diff
@@ -19,3 +20,67 @@ pub trait Migration {
         None
     }
 }
+
+/// A migration name
+#[allow(missing_debug_implementations)]
+#[derive(Clone, Copy)]
+pub struct MigrationName<'a> {
+    /// Wraps around a migration
+    pub migration: &'a Migration,
+}
+
+/// Get the name of a migration
```

Comment:
Can we provide more information than just repeating the function/argument names? Maybe at least a usage example?